api/player.go

```diff
@@ -126,19 +126,18 @@ func (p *Players) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // GET /api/player/list.json
 func (p *Players) ServeList(w http.ResponseWriter, r *http.Request) {
-	p.mutex.Lock()
```

Comment:
for consistency i like the defer syntax; nitpick

Reply:
the smaller the critical area the better \:)